explainer.md

```diff
@@ -88,6 +89,21 @@ Items that are designed to be purchased once and last permanently in the user’
 itemService.acknowledge(purchaseToken, 'onetime');
 ```
 
+### Checking existing purchases
+
+The `listPurchases` method allows a client to get a list of items that are currently owned or purchased by the user. This may be necessary to check for entitlements (eg. whether a subscription, promotional code, or permanent upgrade is active) or to recover from network interruptions during a purchase (e.g. item is purchased but not yet acknowledged).
+
+
+```js
+purchases = await itemService.listPurchases();
+for (p in purchases) {
```

Comment:
s/`in`/`of`

See [Difference between `for...of` and `for...in`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Difference_between_for...of_and_for...in)

Reply:
Ah, I copied the example above. Switched them both to `for ... of`